libs/sdk-ui-pivot/src/impl/agGridDataSource.ts

```diff
@@ -216,3 +231,33 @@ function isSortedByFirstAttribute(headers: TableHeaders, sortModel: ISortModelIt
 
     return headers.allHeaders[0].field === sortModel[0].colId;
 }
+
+function isDataViewSortedByFirstAttribute(dv: DataViewFacade): boolean {
+    const { sortBy } = dv.definition;
+    if (!sortBy || !sortBy.length) {
+        // this is somewhat dangerous assumption: no explicit sort == sorted by first col
+        //  (bear backend behaves thusly)
+        return true;
+    }
+
+    if (sortBy.length > 1) {
+        return false;
+    }
+
+    const firstSortBy = sortBy[0];
+    if (!isAttributeSort(firstSortBy)) {
+        return false;
+    }
+
+    const attributeBucket = dv.def().bucket(BucketNames.ATTRIBUTE);
+    if (!attributeBucket) {
+        return false;
+    }
+
+    const firstAttribute = bucketAttributes(attributeBucket)[0];
```

Comment:
you can do bucketAttribute(bucket) => with default predicate this returns first attr or undefined